packages/generators/init-template/svelte/webpack.config.js

```diff
@@ -0,0 +1,62 @@
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const path = require("path");
+
+const mode = process.env.NODE_ENV || "development";
+const isProduction = mode === "production";
+
+module.exports = {
+    entry: {
+        "build/bundle": ["./src/main.js"],
+    },
+    resolve: {
+        alias: {
+            svelte: path.dirname(require.resolve("svelte/package.json")),
+        },
+        extensions: [".mjs", ".js", ".svelte"],
+        mainFields: ["svelte", "browser", "module", "main"],
+    },
+    output: {
+        path: path.join(__dirname, "public"),
+        filename: "[name].js",
+        chunkFilename: "[name].[id].js",
+    },
+    module: {
+        rules: [
+            {
+                test: /\.svelte$/,
+                use: {
+                    loader: "svelte-loader",
+                    options: {
+                        compilerOptions: {
+                            dev: !isProduction,
+                        },
+                        emitCss: isProduction,
+                        hotReload: !isProduction,
+                    },
+                },
+            },
+            {
+                test: /\.css$/,
```

Comment:
we should also add support for postcss, sass, etc here.

Reply:
I agree, ideally we should extend default template and add custom loaders/plugins